tests/unittests/benchmark/test_benchmark_client.py

```diff
@@ -306,3 +307,60 @@ def insert_race_condition(*args, **kwargs):
 
             assert bm.configuration == benchmark_config
             assert count_benchmarks() == 1
+
+    def test_create_with_executor(self, benchmark_config, benchmark_config_py):
+
+        with OrionState():
+            config = copy.deepcopy(benchmark_config_py)
+            bm1 = get_or_create_benchmark(**config)
+
+            assert bm1.configuration == benchmark_config
+            assert bm1.executor.n_workers == orion.core.config.worker.n_workers
+
+            executor = Joblib(n_workers=2, backend="threading")
+            config["executor"] = executor
+            bm2 = get_or_create_benchmark(**config)
+
+            assert bm2.configuration == benchmark_config
+            assert bm2.executor.n_workers == executor.n_workers
+
+    def test_experiments_parallel(
+        self, benchmark_config_py, benchmark_algorithms, experiment_config, monkeypatch
+    ):
+        from orion.benchmark import Study
+        from orion.testing import create_experiment
+
+        def foo(x):
+            return [dict(name="result", type="objective", value=x * 2)]
+
+        def optimize(*args, **kwargs):
+            optimize.count += 1
+            return 1
+
+        with create_experiment(
+            exp_config=experiment_config, trial_config={}, statuses=[]
```

Comment:
Do we need to pre-fill the DB with a specific experiment config? Could be empty otherwise to keep it simple.

Reply:
You mean experiment_config with only a name and keep all the others empty?

Reply:
I mean using OrionState and passing no configuration at all.

Reply:
okay, this is how it looks like now :)